apps/video/js/video.js

```diff
@@ -1386,8 +1386,9 @@ function showThrobber() {
 }
 
 function hideThrobber() {
-  dom.throbber.classList.add('hidden');
   dom.throbber.classList.remove('throb');
+  // setTimeout is a workaround for Bug 1166500
+  setTimeout(function() { dom.throbber.classList.add('hidden'); }, 100);
```

Comment:
Yes, I double-checked that. On tablets, and maybe in the view activity for urls instead of blobs we might have this bug in video. But I'm not going to try to address those cases here.